From http://svn.gnome.org/viewcvs/gnome-menus/trunk/layout/applications.menu?view=patch&r1=666&r2=702&pathrev=702
Fixes the "Other" submenu, as Application is not a valid category in desktop files, so we shouldn't require it.
Also exclude screensavers because they'd show up otherwise combined with the previous change.
This makes all menu entries, that don't match any other category, properly show up in the "Other" submenu.

--- trunk/layout/applications.menu	2007/05/01 10:11:09	666
+++ trunk/layout/applications.menu	2007/06/26 21:20:55	702
@@ -143,9 +143,9 @@
     <OnlyUnallocated/>
     <Include>
       <And>
-        <Category>Application</Category>
         <Not><Category>Core</Category></Not>
         <Not><Category>Settings</Category></Not>
+        <Not><Category>Screensaver</Category></Not>
       </And>
     </Include>
   </Menu> <!-- End Other -->
